src/com/google/javascript/jscomp/gwt/client/GwtRunner.java

```diff
@@ -104,13 +104,15 @@
     String[] entryPoint;
     String env;
     boolean exportLocalPropertyDefinitions;
+    Object[] externs;
```

Comment:
Why is `externs` typed as `Object[]` when `js` is `String[]`?

Reply:
Unfortunately, past versions of the GWT compiler handled externs differently - users were expected to pass an array of objects:

```json
[{
  path: '/path/to/extern.js',
  src: 'function foo() {}'
}]
```

In order for this not to be a breaking change, the parsing handles either an array of strings or an array of objects.

See https://github.com/google/closure-compiler/pull/3036/files#diff-0420e060726b261dbc58a5ca6e98613aR704

Reply:
OK, passed that info on to internal reviewer.
I think this will land tomorrow.